core/node/state_keeper/src/metrics.rs

```diff
@@ -242,6 +244,10 @@ impl TxAggregationMetrics {
     pub fn l2_block_reason_inc(&self, reason: &L2BlockSealReason) {
         self.l2_block_reason[reason].inc();
     }
+
+    pub fn record_criterion_capacity(&self, criterion: &'static str, value: f64) {
+        self.criterion_capacity_filled[&criterion].observe(value)
+    }
```

Comment:
Could it make sense to log these values as well in case we need more accurate data? Persisted metrics data for histograms is imprecise because of bucketing.

Reply:
I think 10% is precise enough and we don't need to log it

Reply:
OK. Suggested because the additional volume of logs wouldn't be that high (the logs would be produced per batch).